codesign/test/file_codesign_visitor_test.dart

```diff
@@ -1200,6 +1204,10 @@ status: Invalid''',
         ),
       );
       rootDirectory = fileSystem.systemTempDirectory.createTempSync('conductor_codesign');
+
+      await fileSystem.file(appSpecificPasswordFilePath).delete();
```

Comment:
L24 should be made:

```
  late file FileSystem fileSystem;
```

Then the `setUp` should instantiate the filesystem.

Reply:
Done. Yeah this is cleaner. By the way would recreating filesystems be cheaper than recreating files?